monai/transforms/io/array.py

```diff
@@ -33,20 +34,24 @@
 __all__ = ["LoadImage", "SaveImage"]
 
 
-def switch_endianness(data, old, new):
+def switch_endianness(data, new="<"):
     """
-    If any numpy arrays have `old` (e.g., ">"),
-    replace with `new` (e.g., "<").
+    If any numpy arrays have `new` endianness (e.g., "<").
```

Comment:
I think the doc-string is not completed, could you please help add more doc-string about the `sys_native` logic?